front_end/Inbox/Message.js

```diff
@@ -0,0 +1,52 @@
+import React, {Component} from 'react';
+import {
+    Text,
+    TouchableHighlight,
+    View
+} from 'react-native';
+
+import Icon from '../Utility/Icon';
+import styles from '../Static/style';
+import inbox_style from '../Static/inbox_style';
+
+export default class Message extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            name: props.name,
+            message: props.message,
+            time: props.time,
+            chatID: props.chatID
+            //navigation: props.navigation for later
+        };
+        this.onPressMessage = this.onPressMessage.bind(this);
+    }
+
+    onPressMessage() {
```

Comment:
What is this function supposed to do??

Reply:
it is suppose to go to the chat room. I just put a console log just temporarily